src/subcommand/wallet/runics.rs

```diff
@@ -0,0 +1,37 @@
+use super::*;
+
+#[derive(Serialize, Deserialize)]
+pub struct RunicUtxo {
+  pub output: OutPoint,
+  pub rune: SpacedRune,
+  pub rune_balance: u128,
```

Comment:
```suggestion
  pub runes: BTreeMap<SpacedRune, Decimal>,
```
In theory an output can contain multiple runes, so let's make this a map. Have a look at `src/subcommand/wallet/balance.rs` for how to create this struct.